pubspec.yaml

```diff
@@ -1,25 +1,28 @@
 name: form_builder_validators
 description: Form Builder Validators set of validators for FlutterFormBuilder. Provides common validators and a way to make your own.
-version: 9.1.0
+version: 9.2.0
```

Comment:
If the minimum flutter version and dart version will be increased, this is a breaking change and a major increase version